ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+### **Description**
+
+Please describe the issue you are facing or a feature you would like to be added to the SDK.
+
+<!-- If making a feature request, remove the below information -->
+### **Repro Steps**
+
+Please list the steps used to reproduce your issue.
+
+1.
+2.
+
+### **Details**
+
+1. Which SDK version are you using?
+    - e.g. 1.5.0
+2. Which OS version did you experience the issue on?
+    - e.g. iOS 11
+3. Which Cocoapods version are you using (run `pod --version`)?
+    - e.g. 1.4.0
+4. What device version did you see this error on?  Were you using an emulator or a physical device?
+    - e.g. iPhone X physical device, iPhone 8 emulator
+5. What language are you using?
+    - [ ] Objective C
+    - [ ] Swift
+6. What third party libraries are you using?
+    - example
```

Comment:
The "example" alone feels weird here, I think we either remove it or find real examples